tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -166,7 +166,7 @@ var actionInvokeCmd = &cobra.Command{
     if err != nil {
       whiskErr, isWhiskErr := err.(*whisk.WskError)
 
-      if (isWhiskErr && whiskErr.ApplicationError != true) || !isWhiskErr {
+      if (isWhiskErr && whiskErr.ApplicationError != true && whiskErr.ExitCode != 202) || !isWhiskErr {
```

Comment:
Special handling of 202 response codes should only happen on blocking action invocations